source/_integrations/eway.markdown

```
@@ -0,0 +1,193 @@
+(kWh)
+- **Error Code**: Device error code (if any)
+- **Working Duration**: Total working time 
+(seconds)
+
+## Requirements
+
+- Home Assistant 2023.1+
+- aioeway library (automatically installed)
+- MQTT broker
+
+## Installation
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)